python/private/repo_utils.bzl

```diff
@@ -262,12 +341,50 @@ def _outputs_to_str(result):
             lines.append("<{} empty>".format(name))
     return "\n".join(lines)
 
+def _get_platforms_os_name(rctx):
```

Comment:
Note to myself, maybe I should have a followup to use this in `select_requirement` function. Might be useful to deduplicate.